llama_index/output_parsers/selection.py

```diff
@@ -45,7 +46,15 @@ class Answer(DataClassJsonMixin):
 
 
 class SelectionOutputParser(BaseOutputParser):
+    def _marshal_to_json(self, output: str) -> str:
+        "llm output to json friendly"
+        output = output.strip()
+        if output.startswith("Output:"):
```

Comment:
hardcoding this is a bit hacky tbh. can we instead do `find` and `rfind` for a `{`/`[` or `}`/`]` characters respectively? that way you prune out any output before or after the actual json